### ISSUE TITLE
Create Atom feed with html content

### ISSUE THREAD

**Issue by chriswill** [dotnet/SyndicationFeedReaderWriter#8]
I want to publish the content of a blog in an Atom feed.

I create my SyndicationItem object and set it's Description property to the html and see that it creates an Atom content element.  So far so good.  However I have two questions:

On Atom fields I've seen in the wild (including your own test atom feed, the content element has an attribute named "type" that I don't see how to set.  This often appears as "xhtml" or "html".  Shouldn't SyndicationItem have a property for an additional class that would allow to set this attribute?

My additional question is how to make the information within the Atom content element to be non-encoded.  My html angle brackets appear as `"&lt;p&gt;"` instead of `<p>`, making my html useless.

thanks

**Comment by drago-draganov** [comment]
IAtomEntry has property ContentType.
If you like to write Atom Text Construct with custom type, you can use AtomFeedWriter.WriteText method.

Regarding the second question:

RFC 4287 section 3.1.1.2

>    If the value of "type" is "html", the content of the Text construct
>    MUST NOT contain child elements and SHOULD be suitable for handling
>    as HTML [HTML].  Any markup within MUST be escaped; for example,
>    "<br>" as "&lt;br>".


You can use xhtml type and it will appear unescaped, wrapped in <div> (according to RFC 4287 section 3.1.1.3). 

  ```
<content type="xhtml">
      <div xmlns="http://www.w3.org/1999/xhtml">
         This is <b>XHTML</b> content.
      </div>
   </content>
```

Note that the current released version 1.0 has a bug #6 dealing with XHTML type content. It was addressed with #7.

**Comment by chriswill** [comment]
Thanks.  I see the way to create the content type in IAtomEntry.  Previously I went down the wrong path because I was using SyndicationItem.

`AtomEntry atomEntry = new AtomEntry();
atomEntry.ContentType = "xhtml";`

Looking forward to the update, and thanks for the library.
